src/app/MessageDef/ArrayBuilder.cpp

```diff
@@ -0,0 +1,48 @@
+/**
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+/**
+ *    @file
+ *      This file defines Array builder in CHIP interaction model
```

Comment:
nit: consider removing the `@file`: it seems obvious a file named `ArrayBuilder.cpp` `defines Array builder` (probably does not define but implementis since it is a cpp file)

Reply:
done